dot/state/grandpa_test.go

```diff
@@ -29,8 +29,13 @@ var (
 )
 
 func TestNewGrandpaStateFromGenesis(t *testing.T) {
+	ctrl := gomock.NewController(t)
+
+	telemetryMock := NewMockTelemetry(ctrl)
+	telemetryMock.EXPECT().SendMessage(gomock.Any()).AnyTimes()
```

Comment:
replace `gomock.Any()` with appropriate types.
replace `.AnyTimes()` with appropriate number of times `.Times(n)`

Reply:
Solved